src/organisms/tables/ComparisonTable/index.js

```diff
@@ -111,9 +111,7 @@ class TableHeader extends React.Component {
           <Col className={styles['col-offset-header']} />
 
           {
-            this.headerToRender.map((child, idx) =>
-              <Col key={idx} className={styles['col-offset-header-cells']}>{ child }</Col>
-            )
+            this.headerToRender.map((child, idx) => <Col key={idx} className={styles['col-offset-header-cells']}>{ child }</Col>)
```

Comment:
same here for multi-line arrow functions, I think you can use parens actually for jsx elements

Reply:
changed